ods_ci/build/ldap/ldap.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openldap
+---
+apiVersion: v1
+data:
+  #Password is “adminpassword”
```

Comment:
I think thats ok for PoC but not sure it will be good practice to use hard-coded values of LDAP secrets in general.
let me know wyt

Reply:
For now, let's keep the static creds. If we decided to move to dynamic creds then it should be done on a higher level. That means on "testing group" level and not only on interop level.

Reply:
We can wait until we set up Vault on OpenShift CI.